rotkehlchen/chain/ethereum/adex/adex.py

```diff
@@ -242,36 +286,38 @@ def _get_staking_history(
             adx_staking_details = []
             for adx_staking_balance in adx_staking_balances:
                 # NB: currently it only returns staking details for Tom pool
-                if adx_staking_balance.pool_id == TOM_POOL_ID and tom_pool_incentive is not None:
-                    adx_total_profit = Balance()
-                    dai_total_profit = Balance()
-                    # Add claimed amounts and their historical usd value
-                    for event in address_staking_events[address]:
-                        if isinstance(event, ChannelWithdraw):
-                            if event.token == A_ADX:
-                                adx_total_profit += event.value
-                            elif event.token == A_DAI:
-                                dai_total_profit += event.value
-
-                    # Add unclaimed amounts and their current usd value
-                    adx_total_profit += adx_staking_balance.adx_unclaimed_balance
-                    dai_total_profit += adx_staking_balance.dai_unclaimed_balance
-
-                    pool_staking_detail = ADXStakingDetail(
-                        contract_address=adx_staking_balance.contract_address,
-                        pool_id=adx_staking_balance.pool_id,
-                        pool_name=adx_staking_balance.pool_name,
-                        total_staked_amount=(
-                            tom_pool_incentive.total_staked_amount / ADX_AMOUNT_MANTISSA
-                        ),
-                        apr=tom_pool_incentive.apr,
-                        adx_balance=adx_staking_balance.adx_balance,
-                        adx_unclaimed_balance=adx_staking_balance.adx_unclaimed_balance,
-                        dai_unclaimed_balance=adx_staking_balance.dai_unclaimed_balance,
-                        adx_profit_loss=adx_total_profit,
-                        dai_profit_loss=dai_total_profit,
-                    )
-                    adx_staking_details.append(pool_staking_detail)
+                if adx_staking_balance.pool_id != TOM_POOL_ID:
+                    continue
```

Comment:
perhaps instead ofthe comment above, log a warning here with details for us to pick it up from user logs to know we need to add support for something?

Reply:
True